ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Model.hs

```diff
@@ -80,6 +84,7 @@ data Model blk = Model {
     , currentLedger :: ExtLedgerState blk
     , initLedger    :: ExtLedgerState blk
     , iterators     :: Map IteratorId [blk]
+    , invalidBlocks :: Set (Point blk)
```

Comment:
Do we ever expire things from this set in the model? If not will we have any problems with tests matching the model and real version later?

Reply:
I think this question is addressed in the tests `invalidBlockNeverValidatedAgain`